src/pytorch_lightning/CHANGELOG.md

```diff
@@ -4,6 +4,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [1.7.6] - 2022-09-13
+
+### Changed
+
+- When using multiple loggers, by default checkpoints and profiler output now get saved to the log dir of the first logger in the list ([#14325](https://github.com/Lightning-AI/lightning/pull/14325))
```

Comment:
#14325 This has milestone 1.8, it should not be included.

Reply:
Sorry, I'll remove this! :))

Reply:
Done! @awaelchli - I hope it's okay if I remove and amend the commit to the latest commit; https://github.com/Lightning-AI/lightning/pull/14618/commits/2f6bcbc44eb3859a9fa051c7486fde1838711d7d. Please let me know if it's not okay. :)